File: angular-app/src/models/bus-stop.ts
import { Guid } from "guid-typescript";
import { BusStopsOnLine } from './bus-stops-on-line';

export class BusStop {
    public Id: Guid;
    public X: number;
    public Y: number;
    public Name: string;
    public Address: string;
    public BusStopsOnLines: Array<BusStopsOnLine> = new Array();
}

File: WebApp/WebApp/Models/BusStop.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class BusStop
    {
        [Key]
        public Guid Id { get; set; }
        [Required]
        public double X { get; set; }
        [Required]
        public double Y { get; set; }
        public string Name { get; set; }
        public string Address { get; set; }
        public ICollection<BusStopsOnLine> BusStopsOnLines { get; set; }


    }
}